string table_name
string start_date 
string end_date
int64 offset
int64 length
---
string data
 
